lib/server.ts

```diff
@@ -28,10 +28,46 @@ const SOCIAL_CONTRACT = {
  * @param opts DevOptions
  * @returns http server
  */
-export function startDevServer(devJsonPath: string, opts: DevOptions): http.Server {
+export function startDevServer(srcs: string[], dists: string[], devJsonPath: string, opts: DevOptions): http.Server {
   const app = createApp(devJsonPath, opts);
   const server = http.createServer(app);
-  startServer(server, opts);
+  startServer(server, opts, () => {
+    const postData = JSON.stringify({srcs: srcs.map((src) => path.resolve(src)), dists: dists.map((dist) => path.resolve(dist))});
+    const options = {
+      hostname: '127.0.0.1',
+      port: opts.port,
+      path: `/api/apps`,
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Content-Length': Buffer.byteLength(postData),
+      },
+    };
+
+    const req = http.request(options, (res) => {
+      console.log(`STATUS: ${res.statusCode}`);
```

Comment:
We're in the scenario that a dev server has already been started.

Let's log something better using Logger, something like

log.info(`${srcs} have been added to existing dev server and changes will be watched.`)

And then it seems that this terminal can be freed up -- I tested killing it and keeping the original and it still works. You could exit, or just give a log that says "You can close this terminal"